saleor/graphql/schema.graphql

```diff
@@ -4712,7 +4712,7 @@ type PageTranslatableContent implements Node {
 type PageTranslate {
   translationErrors: [TranslationError!]! @deprecated(reason: "This field will be removed in Saleor 4.0. Use `errors` field instead.")
   errors: [TranslationError!]!
-  page: PageTranslatableContent
+  page: Page
```

Comment:
Looks like unexpected change.

Reply:
This change comes from here https://github.com/mirumee/saleor/blob/2e7b5aa1b312e1792ad3ff7930e0217d60be5629/saleor/graphql/translations/mutations.py#L44
I'm not sure why that is the case though, because it seems to me that it should've been "Page" already before, since that's the value of `page_models.Page._meta.object_name`.
I think that it should stay as `Page` in the schema, since if I'm not mistaken, we want to point to the actual model here.

Reply:
Not sure if we should change the schema in that way. 
@maarcingebala are you able to help here? I am not sure if the PageTranslatableContent is something expected here.

Reply:
It looks like in other translation types like this (e.g. ProductTranslation) we don't use the TranslatableContent type, so it seems that it should be `page: Page` before 🤔

Reply:
Yes, it looks like a bug and it should be a `page`. 
This type is only used in:
`pageTranslate(id: ID!, input: PageTranslationInput!, languageCode: LanguageCodeEnum!): PageTranslate`
So I assume that the dashboard also uses `PageTranslatableContent` instead of `Page`.
Not sure what should we do with this now. I am worrying that this change can break any integration which operates on page: PageTranslatableContent. 
cc @maarcingebala

Reply:
@maarcingebala @korycins 

- django-graphene keeps a registry (1:1 function) between graphene.DjangoObjectTypes and django.db.Model
- this 1:1 mapping is crucial - DjangoObjectType automatically converts from model instances returned by resolvers to object types
- there are several models with a duplicated ObjectType (I can send you a list if you ping me on Slack)
- if there are multiple ObjectTypes with the same model, the latest one to be imported ends up in the registry

This is an example commit, that "reverses" this change in schema by delaying the import of the "Page object type":
https://github.com/mirumee/saleor/commit/8f6754666292494cfd066c7e5087f6e1f5df7d47

Reply:
We want to get rid of `django-graphene` as it is blocking us from using `asyncio`. Can we change both to regular `ObjectType`s?

Reply:
@patrys we'll implementing the new `ShippingMethod` as a plain `ObjectType`. I'll let you know how complex does it make things.

Reply:
Not sure if @patrys  and @maarcingebala  agree with me but I wouldn't change the returned type for the page (from `PageTranslatableContent` to `Page`).  I agree that we need to change it as it is a bug but I would like to do that in separate PR after we will confirm that we won't break any integration/storefront.

Reply:
@maarcingebala @patrys @korycins 

For now what we'll do is:
```
    @classmethod
    def get_type_for_model(cls):
        return PageTranslatableContent
```

This ensures that the type in `PageTranslate` won't change in this PR. This could be even a long term fix for Saleor's `ModelMutation` as we don't seem to be using other Django-graphene features that touch the registry. It should be pretty straightforward to implement project-wise as we can raise NotImplemented in `BaseTranslateMutation` and fix tests